Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: clean clean_remote pull_remote installdeps_remote deploy
 
 USER=mulungwishi
-HOST=107.155.72.115
+HOST=104.251.216.252
```

Comment:
@admiral96 then maybe the lesson is not to hardcode IPs, if you can use the DNS name